ios/purchases_flutter.podspec

```diff
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'Purchases', '~> 3.0.1'
+  s.dependency 'Purchases', '~> 3.2.1'
 
   s.ios.deployment_target = '9.0'
```

Comment:
should this have a mac os deployment target?

Reply:
Flutter is not compatible with mac os yet